### ISSUE TITLE
[Org sign up] The field is required but is missing required indicator

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5842]
## Area or Component
org sign up

## Quick Summary
The "Sign up for SimpleReport" page on `/sign-up` has a form group that is required but there is no indication that it is required

<img width="652" alt="Screenshot 2023-05-19 at 13 27 25" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/d18f2dd3-0363-4899-a31e-a796340b838b">

## Accessibility Violation

https://cdc-devtools.dequecloud.com/issues/f6dc1532-146a-434e-968a-34dbc8acae45

## Action Requested

~~Deque is suggesting we add `aria-required` but since it is a form group I suggest we add the text "(Required)" or "*" to the form group's legend and remove the `usa-sr-only` from the `<legend>` so that it is visible to every user.~~

Change the Continue button to be disabled until one of the three options are selected

**Open question**
I am wondering if we do this change we should put "To get you to the right place, tell us a little about yourself." in the legend instead of the `<p>` above?

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by emyl3** [comment]
@BobanL found info on how to handle single input forms where the only input is required: https://dequeuniversity.com/class/wcag-conformance/interpreting/wcag/a/3.3.2/3.3.2.c



![Image](https://github.com/CDCgov/prime-simplereport/assets/20211771/536cf717-7ae7-4117-aa03-8910efd46a06)
